app/containers/TwoFactor/styles.js

```diff
@@ -14,7 +14,8 @@ export default StyleSheet.create({
 		borderRadius: 4
 	},
 	title: {
-		fontSize: 14,
+		fontSize: 16,
```

Comment:
I don't disagree on these changes, but they are unrelated to this PR.
This PR fixes a focus issue.
Please open another PR fixing this `title` style mentioning before and after your changes.
Thanks.

Reply:
Done!